src/uct/tcp/tcp_sockcm.h

```diff
@@ -7,30 +7,39 @@
 #include <uct/base/uct_cm.h>
 
 
+typedef struct uct_tcp_sockcm_ep   uct_tcp_sockcm_ep_t;
+
+
 typedef struct uct_tcp_sa_arg {
-    int             fd;
-    ucs_list_link_t list;
+    int                 fd;
```

Comment:
`uct_tcp_sa_arg_t::fd` == `uct_tcp_sockcm_ep_t::fd`
do we really need to have duplicated information here?
`fd` can be taken from `ep`

Reply:
since the ep on the server side isn't implemented yet, i cant save this fd on the server's ep because it doesn't exist yet - https://github.com/openucx/ucx/pull/4611/files#diff-5e1282b3d909dc0b302fbd0db6de3c7aR61
i'll unite when the server's ep is done.

Reply:
> since the ep on the server side isn't implemented yet, i cant save this fd on the server's ep because it doesn't exist yet - https://github.com/openucx/ucx/pull/4611/files#diff-5e1282b3d909dc0b302fbd0db6de3c7aR61
> i'll unite when the server's ep is done.

ok, then makes sense to me